src/main/java/com/krafit/financialcalcs/LoanCalculatorApplication/LoanCalculator.java

```diff
@@ -0,0 +1,14 @@
+package com.krafit.financialcalcs.LoanCalculatorApplication;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+public class LoanCalculator {
+    int principalAmount;
+    int tenureOfMonths;
```

Comment:
can you change this to tenureInMonths

Reply:
changes done . changed to tenureInMonths